OpenRA.Game/World.cs

```diff
@@ -366,9 +371,9 @@ public int SyncHash()
 						ret += n++ * (int)(1 + actor.ActorID) * syncHash.Hash;
 
 				// TODO: don't go over all effects
```

Comment:
Removed. I actually had this in mind as a main goal. Weird that I forgot to erase it.